lib/matplotlib/figure.py

```diff
@@ -2059,6 +2059,16 @@ def savefig(self, fname, *, frameon=None, transparent=None, **kwargs):
             A list of extra artists that will be considered when the
             tight bbox is calculated.
 
+        metadata : dict, optional
+            Key/value pairs to store in the image metadata. The supported keys
+            and defaults depend on the image format and backend:
+
+            - 'png' with Agg backend: See the parameter ``metadata`` of
+              `~.FigureCanvasAgg.print_png`.
+            - 'pdf' with pdf backend: See the parameter ``metadata`` of
+              `.PdfPages`.
```

Comment:
This needs to specify between `matplotlib.backends.backend_pdf.PdfPages` and `matplotlib.backends.backend_pgf.PdfPages`  I wasn't sure if you are OK w/ pushing to your PRs - if so next time I'll just push minor changes like this...

Reply:
Not sure I understand the differences between pdf and pgf. You are welcome to push to my PRs.

Reply:
Pgf needs tikz and a latex compiler to produce a PDF using the pgf file.  I had no idea that it had a PdfPages method.

Reply:
Ok. I've no linked to `backend_pdf.PdfPages`, which was my original intention.